### ISSUE TITLE
Install path on OS X

### ISSUE THREAD

**Issue by avluis** [CANBus-Triple/CANBus-Triple#6]
According to the instructions I am supposed to clone into the Sketchbook or hardware folder but at least on OS X, it will not load the files as the repo is currently set up. This worked for me though:

![screenshot 2015-03-03 10 42 15](https://cloud.githubusercontent.com/assets/875593/6467208/5c8a8edc-c192-11e4-91b8-063382f5a8c1.png)

This seems to be a hit or miss with similar board variants when it comes to the Arduino IDE, but at least it works for now.

![screenshot 2015-03-03 10 45 54](https://cloud.githubusercontent.com/assets/875593/6467235/7d10d684-c192-11e4-9d92-a2b2568daedc.png)

**Comment by etx** [comment]
You need to clone it into [Your Sketchbook folder]\hardware

Make the hardware folder in your sketchbook folder as it probably does not exist. If you got Arduino to find the files elsewhere that is probably fine too.

**Comment by avluis** [comment]
Update - double checked my Arduino install - ended up looking at files from 1.0.6 instead of 1.6.
Something is amiss on my system since this is not the first time that I have had older software linger.
Seems to be an issue with my backup software (I am using the latest beta for OS X 10).

I was able to get Arduino 1.6 IDE installed and the instructions are clearly built for that in mind since it worked right away once my issue on my end was fixed.
